ui/scripts/configuration.js

```diff
@@ -2766,10 +2776,13 @@
                                                     $form.find('.form-item[rel=isPersistent]').find('input[type=checkbox]').attr("disabled", "disabled");
 
 
-                                                } else { //$(this).val() == "Isolated"
+                                                } else if ($(this).val() == "Isolated") {
```

Comment:
Can both the cases for L2 and Isolated be collapsed, as both are using same code?